style.css

```diff
@@ -367,3 +367,152 @@ h4 {
 
   color: #fff;
 }
+
+.form {
+  display: flex;
+}
+
+.form img {
+  position: absolute;
+  width: 74px;
+  height: 74.02px;
+  left: 0;
+  top: 4732px;
+}
+
+.form h1 {
+  position: absolute;
+  width: 249px;
+  height: 50px;
+  left: 16px;
+  top: 4770px;
+  font-family: "Playfair Display", "Trebuchet MS", "Lucida Sans Unicode", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+
+  /* identical to box height, or 50px */
+  color: #1a2236;
+}
```

Comment:
[OPTIONAL] I'm not sure why no other reviewer told you this so far. I've noticed that you used a fixed width for your whole project. Please take into account that, although this is the width displayed on the design, you shouldn't use a fixed width for your projects (it is just the width of the device used for the screenshot). I will explain: 

The idea of Responsive Web Design is that your webpage should adapt itself to any device/window size that opens it. In other words, it is responsive. This means that using a fixed width for your sections will break this responsiveness, causing a static view to be formed. So, what can you do about it? Simple, use percentages! Instead of `width: 327px`, use `width: 100%`. This will make your project stretches to any window size that you can imagine 😉. If you need a smaller section, use a lower percentage. If you really need to use a fixed width, keep in mind that it may break the responsiveness of your webpage 😄! 

_But, my mobile version looks horrible stretched to a big window 😢._ 

Don't worry about this! There is a thing called [media query](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries) that will allow you to add different styling to different window sizes in the future. Then, you can edit a mobile version and a desktop version for your apps, depending on the window size 😄. But for now, you should focus on the mobile version 😉. 

With a fixed width, even on mobile screens, it doesn't fit correctly. Check your project on an iPhone SE: 
project image on an iPhone XR
![image](https://user-images.githubusercontent.com/42799579/207973867-c1ebfe71-1cec-4b15-857e-65a3e8109be9.png)

If you are interested to learn more about it, check [this link](https://www.w3schools.com/html/html_responsive.asp) and this [youtube video](https://www.youtube.com/watch?v=VQraviuwbzU). 

Good luck and keep coding 😃!

Reply:
Ok @Gopxfs , I am not going to send this change now. But I will implement this on the next "sprint". Thanks for your suggestions, these are really helpful to me.